docs/real-user-metrics.md

```diff
@@ -1,24 +1,32 @@
 
 ## Loading
 
 The LUX scripts should only be loaded when a user has opted into usage tracking. This is done using the `rum-loader` script, which checks if the `usage` cookie is `true` and then loads the required scripts. This is already part of the [Public Layout component].
 
-## Custom changes when updating LUX
+## How it works
 
-The scripts for the real user metrics are loaded from our servers - this allows us to know what is contained within the scripts and reduce the risk of unwelcome things being added to GOV.UK.
+The scripts for the real user metrics are loaded from our servers - this allows us to know what is contained within the scripts and reduce the risk of unwelcome things being added to GOV.UK. Normally LUX is loaded from SpeedCurve's servers, so our setup is unusual.
 
-Because of this, the scripts are audited before being updated and then two extra lines are added to make sure it works correctly. These two lines set the customer ID and turn on beacon mode.
+### The scripts
+
+There are three scripts involved.
+
+- [the loader](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/rum-loader.js) is a script that we wrote. If cookies have been accepted, it [finds the LUX script](https://github.com/alphagov/govuk_publishing_components/commit/0edd70614589d5fbd2a619483d3a63272714b384#diff-68c3e9ad18ca1324acd992819b248cd5b0e3e336f923a972b98ab09e8a06bd9dR58), attaches it to the DOM and executes it. If cookies haven't been accepted, it sets a listener for the `cookie-consent` event.
+- [the measurer](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/vendor/lux/lux-measurer.js) measures performance from the start of page load, but doesn't report anything. This file shouldn't ever change. The source is from the RUM dashboard, in Settings, Edit RUM, at the end. We don't use this in the way SpeedCurve recommends, because we load the scripts ourselves.
+- [the reporter](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/vendor/lux/lux-reporter.js) is the script that the loader pulls down and attaches to the DOM. This file is [compiled as a separate asset](https://github.com/alphagov/govuk_publishing_components/commit/90a9909c58aeec178563a54169c428a738a975ac#diff-545791bbf923733e97dc2063b8df472171f16bd844a6f75ee1f51f695e65b92bR12) and isn't included until users consent to cookies.
+
+The reporter is the file that SpeedCurve occasionally changes (in SpeedCurve's repo it's called `lux.js`). Because we load our own modified copy of this script we currently have to manually download and update it. The script is audited before being updated and then two extra lines are added to make sure it works correctly. These two lines set the customer ID and turn on beacon mode.
```

Comment:
```suggestion
The reporter is the file that SpeedCurve occasionally changes (in SpeedCurve's repo it's called `lux.js`). Because we load our own modified copy of this script we currently have to manually download and update it. The script is audited before being updated and then two extra lines are added to make sure it works correctly. These two lines set the customer ID and set the sampling rate.
```

Reply:
Updated, thanks.